nosqlbench/http-jsonapi-crud-basic.yaml

```diff
@@ -1,8 +1,8 @@
-min_version: "4.17.15"
+min_version: "5.17.1"
 
 # nb5 -v run driver=http yaml=http-jsonapi-crud-basic jsonapi_host=my_jsonapi_host auth_token=$AUTH_TOKEN
 
-description: |
+description: >2
```

Comment:
is this change wanted? what s `>2` doing?

Reply:
I think that's a YAML feature, where there's a myriad of ways to specify how text blocks are included (String values, not YAML markup -- often embedded JSON, XML, CSV, shell scripts)
Which can be convenient but also confusing until you learn all of them when someone uses them (at least I find them confusing).